### ISSUE TITLE
forked version

### ISSUE THREAD

**Issue by effzett** [LazyT/obpm#9]
I have a forked version under
https://github.com/effzett/obpm/tree/fz/wishes

where I tried to realize my wishes for obpm. 
Not clean programming but it works to show the point...;-)

**Comment by probonopd** [comment]
Can you explain what your wishes are? Add confidence bands?

**Comment by effzett** [comment]
1) Yes, confidence intervals would be a nice feature.
Specially when they are linked to the displayed/selected time interval.
So you would see better the mean values for selected time intervals.

2) Furthermore the symbols seem too big when you display a bunch of data.
So the size of the symbols should be linked (somehow) to the number of data;
If you have 100 or more data points the size of the symbols should be smaller.
But not too small so that you can click on every data point (in my fork I’ve forgotten this ;-))

3) The printed version has a much too big legend that overlaps too much data.

4) Finally an arrow from the comments in the graphic to the actual data points seems to me very helpful as data grow…
So you could better identify the commented data points.

— 
fz@zenmeister.de <mailto:fz@zenmeister.de>
Frank Zimmermann

**Comment by probonopd** [comment]
Sounds reasonable to me, but imho the confidence intervals should be a opt-in checkbox in order not to confuse elderly users.

**Comment by effzett** [comment]
Agree, as an additionally option (maybe settings?) it’s a good idea.
— 
fz@zenmeister.de <mailto:fz@zenmeister.de>
Frank Zimmermann

**Comment by probonopd** [comment]
Maybe @LazyT can comment which of the points 1-4 above you would be willing to integrate if you would get PRs for them. :-)

**Comment by LazyT** [comment]
Sorry, but there is not time at the moment as you can see at the commits...
